sys/build.sh

```diff
@@ -93,28 +93,28 @@ if [ "${USE_CS5}" = 1 ]; then
 	CFGARG="${CFGARG} --with-capstone5"
 fi
 
-if [ "`uname`" != Darwin -a -n "${PREFIX}" -a "${PREFIX}" != /usr ]; then
+if [ "$(uname)" != Darwin ] && [ -n "${PREFIX}" ] && [ "${PREFIX}" != /usr ]; then
 	CFGARG="${CFGARG} --with-rpath"
 fi
 
-ccache --help > /dev/null 2>&1
+ccache --help > /dev/null
 if [ $? = 0 ]; then
 	[ -z "${CC}" ] && CC=gcc
 	CC="ccache ${CC}"
 	export CC
 fi
 
 # Required to build on FreeBSD
-if [ ! -x /usr/bin/gcc -a -x /usr/bin/cc ]; then
+if [ ! -x /usr/bin/gcc ] && [ -x /usr/bin/cc ]; then
 	export CC=cc
 	export HOST_CC=cc
 fi
 
 # purge first
 if [ "${PREFIX}" != "/usr" ]; then
-	A=`readlink /usr/bin/radare2 2>/dev/null`
+	A=$(readlink /usr/bin/radare2 2>/dev/null)
 	B="${PWD}/binr/radare2/radare2"
-	if [ -n "$A" -a ! -f "$A" ]; then
+	if [ -n "$A" ] && [ ! -f "$A" ]; then
```

Comment:
why this change? -a should be portable

Reply:
It is not - see https://github.com/koalaman/shellcheck/wiki/SC2166

Reply:
i agree with the changes. `&&` is more portable.

Reply:
it's equally portable, the problem only arises when the string starts with a dash, but im fine with the change